packages/indy-vdr/src/dids/__tests__/IndyVdrSovDidResolver.test.ts

```diff
@@ -0,0 +1,127 @@
+import { JsonTransformer } from '@aries-framework/core'
+
+import didSovR1xKJw17sUoXhejEpugMYJFixture from '../../../../core/src/modules/dids/__tests__/__fixtures__/didSovR1xKJw17sUoXhejEpugMYJ.json'
+import didSovWJz9mHyW9BZksioQnRsrAoFixture from '../../../../core/src/modules/dids/__tests__/__fixtures__/didSovWJz9mHyW9BZksioQnRsrAo.json'
+import { parseDid } from '../../../../core/src/modules/dids/domain/parse'
+import { getAgentConfig, getAgentContext, mockFunction, mockProperty } from '../../../../core/tests/helpers'
```

Comment:
Not for now, but we would _really_ benefit from creating a test utils package i think